.travis/custom_release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+set -x
+
+#!/bin/bash
+set -e
+set -x
+
+if [ "${TRAVIS_BRANCH}" = "master" ]
+then
+    for env in ci
+    do
+        echo "PUSHING ${env}-beta & ${env}-stable"
+        rm -rf ./dist/.git
+        .travis/release.sh "${env}-beta"
+        .travis/release.sh "${env}-stable"
```

Comment:
Not sure where this `.travis/release.sh` script is but seems like all the other services are doing the same.

Reply:
Script resides here [https://github.com/RedHatInsights/insights-frontend-builder-common](https://github.com/RedHatInsights/insights-frontend-builder-common)